### DISCUSSION TITLE
Hierachy of classes - electrical energy

### DISCUSSION THREAD

**Discussion by janfeitkenhauer** [openego/oeplatform#1888] [General]
The class "electrical energy" subdivides into three sub-classes:
nuclear
renewable
solar
-> see here: [OEO](https://openenergyplatform.org/ontology/oeo/OEO_00000139/)

Is there a reason why solar is not categorized as renewable electrical energy?

![Screenshot 2024-10-09 electrical energy](https://github.com/user-attachments/assets/05e40117-9b81-41f2-822d-00bf3409123e)

**Comment by stap-m** [comment]
This is a problem of the oeo viewer on OEP, which seems to only show the asserted hierachy of OEO. 
However, in OEO, it is inferred as renewable electrical energy (this is a snapshot of the tool Protege): 
![grafik](https://github.com/user-attachments/assets/62be8544-5657-479b-b6f5-b7ae26e3c9f1)

You can download the inferred OEO hierarchy with the file oeo-closure.owl from OEP. 
@jh-RLI @Ludee is there a plan to integrate the inferred hierarchy into the viewer, e.g. in NFDI?

**Reply by jh-RLI** [reply]
So far there are no plans that explicitly target this problem. There are other efforts to develop ontology viewer tools, maybe we can bring this up there.

I'd have to look at the code again, but I know we're already retrieving the subclasses, but we're not showing the relationships. We're definitely not asking for the equivalent classes. It would be good to have a topic for that. I'll try to remember this when we discuss the updates to the oeo viewer as part of NFDI4Energy.
